src/background.js

```diff
@@ -1129,29 +1107,48 @@ function setupHubLayoutABTest() {
 }
 
 /**
- * Adjust antitracking parameters based on the current state
- * of ABTest and availability of Human Web.
+ * @since 8.5.3
+ *
+ * Setup the Bloom Filter `antitracking_whitelist2` test.
+ * @memberOf Background
  */
-function setupABTest() {
-	const antitrackingConfig = getAntitrackingTestConfig();
-	if (antitrackingConfig && conf.enable_anti_tracking) {
-		if (!conf.enable_human_web) {
-			// force disable anti-tracking telemetry on humanweb opt-out
-			antitrackingConfig.telemetryMode = 0;
-		}
-		Object.keys(antitrackingConfig).forEach((opt) => {
-			const val = antitrackingConfig[opt];
-			log('antitracking', 'set config option', opt, val);
-			antitracking.action('setConfigOption', opt, val);
-		});
-	}
+function setupBloomFilterABTest() {
 	if (abtest.hasTest('antitracking_whitelist2')) {
 		cliqz.prefs.set('attrackBloomFilter', false);
```

Comment:
We don't use this setting anymore. This test can be removed.

Reply:
Done